kinetic/distribution.yaml

```diff
@@ -15150,6 +15150,151 @@ repositories:
       url: https://github.com/neufieldrobotics/spinnaker_sdk_camera_driver.git
       version: master
     status: maintained
+  sr_common:
+    doc:
+      type: git
+      url: https://github.com/shadow-robot/sr_common.git
+      version: kinetic-devel
+    release:
+      packages:
+      - sr_common
+      - sr_description
+      - sr_robot_msgs
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/shadow-robot/sr_common.git
```

Comment:
The release repository should be a release repository not the upstream source repository.